src/prometheus_alert_rules/probe_failure.rule

```diff
@@ -0,0 +1,13 @@
+groups:
+  - name: probes
+    rules:
+    - alert: ProbeFailed
+      expr: probe_success{} < 1
+      for: 5m
+      labels:
+        severity: critical
+      annotations:
+        summary: Blackbox Exporter target {{ $labels.probe_target }} has failed for at least 50% of the time over the last ten minutes
```

Comment:
- Summary & description suggests that an `avg` is missing in the `expr`? Also, 5 min or 10 min?
- Sometimes authors add a `severity: warning` e.g. immediately when a probe fails, and then the critical comes when it's persistent/flapping.

Reply:
* Addressed
* Do we have any charms that do this?